core/test/mpi/distributed/preconditioner/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+ginkgo_create_test(schwarz MPI_SIZE 1)
```

Comment:
should it be 1?

Reply:
I think for core we are not using any MPI functions, so it should be fine. We do the same for matrix and other distributed tests as well.